radio/src/strhelpers.cpp

```diff
@@ -294,7 +294,7 @@ char * getSwitchString(char * dest, swsrc_t idx)
 #endif
 
   else if (idx <= SWSRC_LAST_LOGICAL_SWITCH) {
-    strAppendStringWithIndex(s, "L", idx-SWSRC_FIRST_LOGICAL_SWITCH+1);
+    strAppendUnsigned(strAppend(s, "L"), abs(idx-SWSRC_FIRST_LOGICAL_SWITCH+1), 2);
```

Comment:
Yes I used strAppend there because LS is 2 letters long, but now with "L" it's not needed anymore